### PR TITLE
Minor documentation improvements for the newbie

### PR DESCRIPTION
**Pull request by pabloguerrero**
# One-line summary
Minor documentation improvements for the newbie.

> ~Zalando ticket : ARUHA-XXX (only if appropriate)~

## Description
This PR adds corrects some minor typos. Also includes an explicit reference to jacoco so that incompatibilities with the JDK on macOS are avoided.

## Review
- [ ] Tests
- [x] Documentation

## Deployment Notes
n/a

### CODE REVIEW

**Comment by adyach** [comment]
thank you for contribution @pabloguerrero ! LGTM, I have to close and re open PR in order travis picks up changes for build/

**Comment by adyach** [comment]
👍

**Comment by lmontrieux** [comment]
👍

File: docs/_documentation/using_comparison.md [hunk 1]
```diff
@@ -68,18 +68,18 @@ The basic abstraction in SQS is a queue, which is quite different from a Nakadi
...
+- Hermes supports JSON Schema and Avro validation in its schema registry. Nakadi's registry currently only supports JSON Schema, but may support Avro in the future. Hermes does not provide inbuilt event types, whereas Nakadi defines optional types to support data change and business process events, with some uniform fields producers and consumers can coordinate on.
 
-- Hermes allows topics (event types in Nakadi) to be collated into groups that are adminstrated by a single publisher. Consumers access data at a per topic level, the same as Nakadi currently; Nakadi may support multi-topic subscriptions in the future via a subscription API. 
+- Hermes allows topics (event types in Nakadi) to be collated into groups that are administrated by a single publisher. Consumers access data at a per topic level, the same as Nakadi currently; Nakadi may support multi-topic subscriptions in the future via a subscription API.
```
Comment (Kunal-Jha) [inline docs/_documentation/using_comparison.md]: I think the correct term here is `administered`

File: docs/_documentation/using_comparison.md [hunk 2]
```diff
@@ -12,50 +12,50 @@ In this section, we'll look at how Nakadi fits in with the stream broker/process
...
 
 - Kinesis uses a common envelope structure for producing and consuming messages, and does not define any higher level structures beyond that. Payload data is submitted as an opaque base64 blob.
 
-- AWS restrict the number of streams available to an account to quite a low starting number, and messages can be stored for a maximum of 7 days whereas Nakadi can support a large number of event types and the expiration for events is configurable.
+- Amazon restricts the number of streams available to an account to quite a low starting number, and messages can be stored for a maximum of 7 days, whereas Nakadi can support a large number of event types and the expiration for events is configurable.
```
Comment (Kunal-Jha) [inline docs/_documentation/using_comparison.md]: IMO ,`AWS` cannot be replaced by Amazon in this case. They are two different things

File: docs/_documentation/using_comparison.md [hunk 3]
```diff
@@ -12,50 +12,50 @@ In this section, we'll look at how Nakadi fits in with the stream broker/process
...
 
-- Inbuilt event types. Nakadi also optional support for events that describe business processes and data changes. These provide common primitives for event identity, timestamps, causality, operations on data and header propagation. Teams could define their own structures, but there's value in having some basic things that consumers and producers can coordinate on independent of the payload, and which are being checked before being propagated to multiple consumers.
+- Inbuilt event types. Nakadi also optionally supports events that describe business processes and data changes. These provide common primitives for event identity, timestamps, causality, operations on data and header propagation. Teams could define their own structures, but there's value in having some basic things that consumers and producers can coordinate on independently of the payload, and which are being checked before being propagated to multiple consumers.
```
Comment (Kunal-Jha) [inline docs/_documentation/using_comparison.md]: I think what makes more sense to me here is `consumers and producers can coordinate on independently of the payload,` -> `consumers and producers can coordinate on,  independent of the payload,`

File: docs/_documentation/using_comparison.md (hunk 1 above)
Comment (pabloguerrero) [reply docs/_documentation/using_comparison.md]: Sounds right, will adjust.

File: docs/_documentation/using_comparison.md (hunk 2 above)
Comment (pabloguerrero) [reply docs/_documentation/using_comparison.md]: Despite being a service offered in AWS, I think the product is indeed called [`Amazon Kinesis`](https://aws.amazon.com/kinesis/).

**Comment by Kunal-Jha** [comment]
@pabloguerrero Thank you for your contribution.

**Comment by Kunal-Jha** [comment]
👍

**Comment by lmontrieux** [comment]
👍

**Comment by ferbncode** [comment]
:+1:

**Pull request merged**